VMBackup/main/handle.py

```diff
@@ -335,7 +335,7 @@ def daemon():
 
         commandToExecute = para_parser.commandToExecute
         #validate all the required parameter here
-        backup_logger.log(commandToExecute,True)
+        backup_logger.log('The command '+ commandToExecute+ 'is being validated',True)
```

Comment:
' is being validated' instead of 'is being validated' to add space after the commandToExecute content.